docs/source/collect/copy.md

```diff
@@ -36,11 +36,19 @@ metadata:
   name: sample
 spec:
   collectors:
+  #Copies resolv.conf file
     - copy:
-        selector: app=api
+        selector: 
+          - app=api
```

Comment:
Is the switch to an array necessary here?

Reply:
Actually this was not a change I have made. When I used the copy collector it worked using a list under the selector. Looking it closely now, the field is defined as a list of strings in the Copy struct:
```golang
type Copy struct {
	CollectorMeta `json:",inline" yaml:",inline"`
	Name          string   `json:"name,omitempty" yaml:"name,omitempty"`
	Selector      []string `json:"selector" yaml:"selector"`
	Namespace     string   `json:"namespace" yaml:"namespace"`
	ContainerPath string   `json:"containerPath" yaml:"containerPath"`
	ContainerName string   `json:"containerName,omitempty" yaml:"containerName,omitempty"`
}
```

Reply:
Actually I made a PR suggesting the change when that happened to me. I've closed it now.

Reply:
So, just to be clear, are you saying that we don't actually support a direct string here at all, and we've always required it to be an array, and the example was just wrong, so now it's fixed?

Reply:
Yes, exactly! With a string you would get:
Error: failed to parse collector: failed to parse document: v1beta1.SupportBundle.Spec: v1beta1.SupportBundleSpec.Collectors: []*v1beta1.Collect: v1beta1.Collect.Copy: v1beta1.Copy.Selector: []string: ...

Reply:
Perfect, thankjs.